docs/src/operate/c8y/supported-operations.md

```diff
@@ -219,7 +221,7 @@ For e.g: if a child device is incapable of receiving and processing the tedge `r
 but can be restarted directly from the tedge device via some remote restart commands,
 the `c8y_Restart` operation file for the child device can be defined to invoke those remote restart commands.
 
-### List of currently supported operations parameters
+##### List of currently supported operations parameters
 
 * `topic` - The topic on which the operation will be executed.
```

Comment:
`timeout` is missing. I would add here the above note.

Reply:
Resolved by a267f04581ec93b1644543df4d45e998b590aaa8